javalib/src/main/scala/java/lang/Character.scala

```diff
@@ -1662,429 +1963,467 @@ object Character {
+      "countingrodnumerals"                        -> COUNTING_ROD_NUMERALS,
+      "mathematicalalphanumericsymbols"            -> MATHEMATICAL_ALPHANUMERIC_SYMBOLS,
+      "glagoliticsupplement"                       -> GLAGOLITIC_SUPPLEMENT,
+      "nyiakengpuachuehmong"                       -> NYIAKENG_PUACHUE_HMONG,
+      "wancho"                                     -> WANCHO,
+      "mendekikakui"                               -> MENDE_KIKAKUI,
+      "adlam"                                      -> ADLAM,
+      "indicsiyaqnumbers"                          -> INDIC_SIYAQ_NUMBERS,
+      "ottomansiyaqnumbers"                        -> OTTOMAN_SIYAQ_NUMBERS,
+      "arabicmathematicalalphabeticsymbols"        -> ARABIC_MATHEMATICAL_ALPHABETIC_SYMBOLS,
+      "mahjongtiles"                               -> MAHJONG_TILES,
+      "dominotiles"                                -> DOMINO_TILES,
+      "playingcards"                               -> PLAYING_CARDS,
+      "enclosedalphanumericsupplement"             -> ENCLOSED_ALPHANUMERIC_SUPPLEMENT,
+      "enclosedideographicsupplement"              -> ENCLOSED_IDEOGRAPHIC_SUPPLEMENT,
+      "emoticons"                                  -> EMOTICONS,
+      "ornamentaldingbats"                         -> ORNAMENTAL_DINGBATS,
+      "alchemicalsymbols"                          -> ANATOLIAN_HIEROGLYPHS,
+      "geometricshapesextended"                    -> GEOMETRIC_SHAPES_EXTENDED,
+      "supplementalarrowsc"                        -> SUPPLEMENTAL_ARROWS_C,
+      "chesssymbols"                               -> CHESS_SYMBOLS,
+      "cjkunifiedideographsextensionb"             -> CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B,
+      "cjkunifiedideographsextensionc"             -> CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C,
+      "cjkunifiedideographsextensiond"             -> CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D,
+      "cjkunifiedideographsextensione"             -> CJK_UNIFIED_IDEOGRAPHS_EXTENSION_E,
+      "cjkunifiedideographsextensionf"             -> CJK_UNIFIED_IDEOGRAPHS_EXTENSION_F,
+      "cjkcompatibilityideographssupplement"       -> CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT,
+      "tags"                                       -> TAGS,
+      "variationselectorssupplement"               -> VARIATION_SELECTORS_SUPPLEMENT,
+      "supplementaryprivateuseareaa"               -> SUPPLEMENTARY_PRIVATE_USE_AREA_A,
+      "supplementaryprivateuseareab"               -> SUPPLEMENTARY_PRIVATE_USE_AREA_B
+    )
+
+    // From: https://unicode.org/Public/UCD/latest/ucd/Blocks.txt
+    // Note:   When comparing block names, casing, whitespace, hyphens,
+    //         and underbars are ignored.
+    //         For example, "Latin Extended-A" and "latin extended a" are equivalent.
+    //         For more information on the comparison of property values,
+    //           see UAX #44: http://www.unicode.org/reports/tr44/
+    private def toNormalizedName(name: String): String = {
```

Comment:
Thank you for the comment: just-in-time delivery of hard to find pertinent wizardry.
Allows me to ask my next question, just below.